projboard/views.py

```diff
@@ -59,3 +60,27 @@ def home_page(request):
 
 def about_page(request):
     return render(request, 'about/about.html', {})
+
+
+def show_article(request, article_pk):
+
+    # TODO when Authintiaction is implemented => del this line and check wich user is loged in
+    user = User.get_user_by_nickname("User2")
+    article = Article.objects.get(pk=article_pk)
+
+    if request.method == 'POST':
+        try:
+            Like.objects.get(user_id=user.id, article_id=article_pk)
+            Like.delete_like(user_id=user.id, article_id=article_pk)
```

Comment:
Why do you delete like for the article on view method?

Reply:
if user hit like and he has been liked the blog before => that means he wants to undo the like .
else he want to like the blog

Reply:
Her I miss something
If a user would like to select/unselect like it should hit GUI element 
Is'nt it? 
I do not see in this MR nothing in templates related to like, did i miss something?

Reply:
i have make a commet above , see it